device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/Message.java

```diff
@@ -301,7 +301,6 @@ public void setProperty(String name, String value)
             this.properties.remove(messageProperty);
         }
 
-        log.trace("Setting message property");
```

Comment:
This log statement was annoying while I was debugging, and I imagine it would be for users, too